bool success                    # true if the ground has been successfully found, false otherwise
string frame                    # reference frame that ground coordinates refer to
geometry_msgs/Point position    # X,Y,Z location coordinates
